bin/cli.js

```diff
@@ -0,0 +1,71 @@
+#! /usr/bin/env node
+
+var program = require('commander');
+var version = require('../package.json').version;
+var Interchange = require('../interchange');
+var interchange = new Interchange();
+
+program
+  .version(version);
+
+program.command("list")
+    .description("Lists all of the available firmwares")
+    .alias("l")
+    .action(interchange.list_devices);
+
+program.command("ports")
+    .description("Lists all of the attached boards and their ports")
+    .alias("p")
+    .option("-v, --verbose", "List with additional information")
+    .action(interchange.list_ports);
+
+program.command("read")
+    .description("Read firware info from port")
+    .alias("r")
+    .option("-p, --port", "Serial port board is attached to")
+    .action(function (opts) {
+        if (!opts.port) {
+            console.error("Please provide a device path".red);
+            return;
+        }
+
+        interchange.get_firmware_info(opts.port);
+    });
+
+program.command("install [firmware]")
+    .description("Install specified firmware to board")
+    .alias("i")
+    .option("-a, --board <board>", "Type of board/AVR")
+    .option("-p, --port <port>", "Serial port board is attached to")
+    .option("-f, --firmata", "Install firmata version of firmware")
+    .option("-i, --address <address>", "Specify I2C address")
```

Comment:
I didn't have this in my version but we should probably note here that it should be in 0x form so maybe just as an example `eg 0x67`